techniques/general/G167.html

```diff
@@ -10,16 +10,33 @@
       <section class="example">
          <h3>A search function</h3>
          
-            <p>A Web page contains a text field where the user can enter search terms and a button labeled "Search" for performing the search. The button is positioned right after the text field so that it is clear to the user that the text field is where to enter the search term.</p>
+            <p>A web page contains a text field where the user can enter search terms and a button labeled "Search" for performing the search. The button is positioned right after the text field so that it is clear to the user that the text field is where to enter the search term.</p>
             <div>
             	<img alt="A text field with a button positioned to the right demonstrating the use of a button to label a field." src="img/button-as-label.jpg"/>
             </div>
          
       </section>
+
+      <section class="example">
+         <h3>A "reply" or "send message" field</h3>
+         <p>A chat or email web application will often present users a single or multi-line text field used for composing a message.  The field does not have a visible text label, but the purpose of the field - to compose a message to send - is indicated by it being paired with a button represented by a "send message" icon.</p>
+
+        <div>
+          <img src="img/button-send-message.jpg alt="text field containing the text let's go. The field has a button visually overlaying the right side of it. The button is circular and contains an arrow icon that also resembles a paper airplane - which has increasingly gained popularity as a 'send message' identifier. ">
+        </div>
+        
+         
+            <p>A web page contains a text field where the user can enter search terms and a button labeled "Search" for performing the search. The button is positioned right after the text field so that it is clear to the user that the text field is where to enter the search term.</p>
+            <div>
+            	<img alt="A text field with a button positioned to the right demonstrating the use of a button to label a field." src="img/button-as-label.jpg"/>
+            </div>
+         
```

Comment:
```suggestion
         
```

This example is repeated above, so I suggest removing it from here and keeping it only in the section above.